torch/csrc/jit/codegen/cuda/root_domain_map.h

```diff
@@ -378,6 +380,9 @@ class TORCH_CUDA_API ComputeAtRootDomainMapBuilder : private BackwardVisitor {
   DomainKeyMap<DomainKeySet> pending_map_;
   std::unordered_set<Expr*> visited_;
   UnmappableReductionDomains incompatible_domains_;
+
+  // additional parameters
```

Comment:
this comment is not super helpful, can we reword it?

Reply:
Updated, thanks!